Front-End/challenge2/script.js

```diff
@@ -1,7 +1,93 @@
-/*
-TO-DO:
+const calculatorApp = document.querySelectorAll('button');
```

Comment:
Good job overall, I like this approach better than what I recommended in live demo